scripts/Muon_Analysis_2.py

```diff
@@ -41,12 +43,38 @@ def qapp():
     return _app
 
 
-app = qapp()
-try:
-    ex = MuonAnalysis2Gui()
-    ex.resize(700, 700)
-    ex.show()
-    app.exec_()
-except RuntimeError as error:
-    ex = QtGui.QWidget()
-    QtGui.QMessageBox.warning(ex, "Muon Analysis version 2", str(error))
+def main():
+    app = qapp()
```

Comment:
I think we'll need `global muonGUI` here so that the assignment goes back to the global object.

We also need to think about what happens when the GUI gets closed and how to set that back to `None`. Perhaps we do that in `closeEvent` above?